### PR TITLE
Jzy6

### PR DESCRIPTION
**Pull request by zenyui**
Added my profile under "people" and updated README with grading.html link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ Essentials
 Lectures
 --------
 
-1.  23-Aug: [Git, REST, HTTP, AWS](lectures/lecture-01.md)
-2.  30-Sep: Backend Services, CI
-3.  06-Sep: Frontend         
-4.  13-Sep: Docker and Launch
-5.  20-Sep: TBD              
+1.  23-Aug: [Git, REST, HTTP, AWS](course/01-git-ci-rest-aws)
```
Comment (amfleming) [inline README.md]: Why are you changing the lectures?

File: README.md [hunk 2]
```diff
@@ -54,11 +54,12 @@ People
...
-* [Pei-Hsuan Tsai](people/peihsuan-tsai.md) - pt379 - [https://s3.amazonaws.com/iketsai/grading.html](https://s3.amazonaws.com/iketsai/grading.html) 
+* [Pei-Hsuan Tsai](people/peihsuan-tsai.md) - pt379 - [https://s3.amazonaws.com/iketsai/grading.html](https://s3.amazonaws.com/iketsai/grading.html)
 * [Chenyang Wang](people/chenyang-wang.md) - cw823 - [http://ec2-kellywang-666.com.s3-website.us-east-2.amazonaws.com/](http://ec2-kellywang-666.com.s3-website.us-east-2.amazonaws.com/)  
 * [Jialiang Wang](people/jialiang-wang.md) - jw2476 - [https://s3.amazonaws.com/aws-website-cs-fall-xl4ky/grading.html](https://s3.amazonaws.com/aws-website-cs-fall-xl4ky/grading.html)
 * [Jared Wong](people/jared-wong.md) - jmw535 - [https://dxf941vjbdgze.cloudfront.net/grading.html](https://dxf941vjbdgze.cloudfront.net/grading.html)
 * [Rongxin Zhang](people/rongxin-zhang.md)
+* [Zen Yui](people/zen-yui.md) - jzy6 - [http://jzy.io.s3-website-us-east-1.amazonaws.com/grading.html](http://jzy.io.s3-website-us-east-1.amazonaws.com/grading.html)
```
Comment (amfleming) [inline README.md]: Doesn't `Yui` come before `Zhang`?

File: README.md [hunk 3]
```diff
@@ -54,11 +54,12 @@ People
 * [An Le](people/an-le.md) - aql6 - [https://s3-us-west-2.amazonaws.com/anle1/grading.html](https://s3-us-west-2.amazonaws.com/anle1/grading.html)
 * [Fuyuzhen Peng](people/fuyuzhen-peng.md) - fp73 - [http://cs5356.s3-website-us-east-1.amazonaws.com/grading.html](http://cs5356.s3-website-us-east-1.amazonaws.com)
 * [Xialin Shen](people/spark-shen.md) - xs293 - [http://cs5356-xs293.s3-website-us-west-2.amazonaws.com/grading.html](http://cs5356-xs293.s3-website-us-west-2.amazonaws.com/grading.html)
-* [Pei-Hsuan Tsai](people/peihsuan-tsai.md) - pt379 - [https://s3.amazonaws.com/iketsai/grading.html](https://s3.amazonaws.com/iketsai/grading.html) 
+* [Pei-Hsuan Tsai](people/peihsuan-tsai.md) - pt379 - [https://s3.amazonaws.com/iketsai/grading.html](https://s3.amazonaws.com/iketsai/grading.html)
```
Comment (amfleming) [inline README.md]: Please don't modify other ppl's entries

**Review by amfleming — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zenyui) [reply README.md]: not on purpose!  will fix.

**Review by zenyui — COMMENTED** [review]
re-pulled your master and made my changes.  This PR should satisfy your prior comments.  Thanks.

**Pull request merged**